Lib9c/Miner.cs

```diff
@@ -21,7 +21,9 @@ public class Miner
 
         public Address Address { get; }
 
-        public async Task<Block<PolymorphicAction<ActionBase>>> MineBlockAsync(CancellationToken cancellationToken)
+        public async Task<Block<PolymorphicAction<ActionBase>>> MineBlockAsync(
+            CancellationToken cancellationToken,
+            int txBatchSize)
```

Comment:
How about switching positions? (many of async methods receive `CancellationToken` as the last parameter)

```suggestion
            int txBatchSize,
            CancellationToken cancellationToken)
```

Reply:
Amended! [`da3438c`](https://github.com/planetarium/lib9c/pull/161/commits/da3438c77a7375cc91399007b8bb58eca48fbcc0)